spec/statesman/machine_spec.rb

```diff
@@ -374,6 +374,47 @@ def after_initialize; end
     end
   end
 
+  describe "#in_state?" do
+    before do
+      machine.class_eval do
+        state :x, initial: true
+        state :y
+        transition from: :x, to: :y
+      end
+    end
+
+    let(:instance) { machine.new(my_model) }
+    subject { instance.in_state?(state) }
+    before { instance.transition_to!(:y) }
+
+    context "when machine is in given state" do
+      let(:state) { "y" }
+      it { is_expected.to eq(true) }
+    end
+
+    context "when machine is not in given state" do
+      let(:state) { "x" }
+      it { is_expected.to eq(false) }
+    end
+
+    context "when given a symbol" do
+      let(:state) { :y }
+      it { is_expected.to eq(true) }
+    end
+
+    context "when given multiple states" do
+      context "when one of the states is the current state" do
+        subject { instance.in_state?(:x, :y) }
+        it { is_expected.to eq(true) }
+      end
+
+      context "when none of the states are the current state" do
+        subject { instance.in_state?(:x, :z) }
+        it { is_expected.to eq(false) }
+      end
```

Comment:
Opinions on passing an array vs multiple args? Prefer one over the other, or both?